src/Foundatio.Kafka/Messaging/KafkaMessageBusOptions.cs

```diff
@@ -35,15 +33,25 @@ public class KafkaMessageBusOptions : SharedMessageBusOptions {
 
     public SecurityProtocol? SecurityProtocol { get; set; }
 
-    public int TopicNumberOfPartitions { get; set; } = -1;
+    public int TopicNumberOfPartitions {
+        get;
+        set;
+    } = -1;
 
-    public short TopicReplicationFactor { get; set; } = -1;
+    public short TopicReplicationFactor {
+        get;
+        set;
+    } = -1;
```

Comment:
Let's make these nullable and then conditionally set them. can we also put all properties get;set; on same line